packages/app/src/app/pages/Dashboard/Sidebar/index.js

```diff
@@ -7,6 +7,7 @@ import Input from '@codesandbox/common/lib/components/Input';
 import { Button } from '@codesandbox/common/lib/components/Button';
 import TimeIcon from 'react-icons/lib/md/access-time';
 import PeopleIcon from 'react-icons/lib/md/people';
+import SaveIcon from 'react-icons/lib/md/save';
```

Comment:
We probably need to change this icon to the one Bas made.

Reply:
I don't have it :(

Reply:
I don't have it :(

Reply:
Changed